src/Tasks/Exec.cs

```diff
@@ -410,7 +410,7 @@ protected override void LogEventsFromTextOutput(string singleLine, MessageImport
 
             if (ConsoleToMSBuild)
             {
-                string trimmedTextLine = singleLine.Trim();
+                string trimmedTextLine = singleLine.TrimEnd();
```

Comment:
Writing my thoughts :) 
Was it considered to be put under change wave or it is not this kind of change ?

Reply:
Good question. @JanKrivanek, @ladipro, @rokonec what do you think?

Reply:
From our conversation with @rokonec, this change should be under change wave as some people can rely on the buggy behavior of the output lines being trimmed. I will make the changes to the code. Thanks Farhad!